### PR TITLE
add home work 2

### PR DESCRIPTION
**Pull request by lesiaturuta**

### CODE REVIEW

File: homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                        Technologies like HTML, CSS, JavaScript, and Git version control system.</p>
+                    <p class="text">This page was developed during the course
+                        <a class="link--red" href="https://www.mastersacademy.education/frontend-for-beginners-it"
+                            target="_blank">'Frontend for beginners'
+                            from Masters Academy in 2023.</a>
+                    </p>
+                    <p class="text">This is a social project from MOCG company where I got an opportunity to work with
+                        Frontend
+                        mentors and to create my own small project for the portfolio.</p>
+                    <p class="text">You can contact me via [Links to Socials Nets] and/or check out my GitHub.</p>
```
Comment (vadym-zinchenko-moc) [inline homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html:56]: pls add your social nets and git hub links

File: homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    <body>
+        <header class="header">
+            <nav class="nav justify__content--center">
+                <div class="container justify__content--between">
+                    <div class="nav__logo">
+                        <img class="logo__icon" src="./assets/logo-icon.svg" alt="logo">
+                        <img class="logo__title" src="./assets/name-icon.svg" alt="name">
+                    </div>
+                    <ul class="nav__menu">
```
Comment (vadym-zinchenko-moc) [inline homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html:24]: I see big distance between About and Pokemon links.  And I think header can have full width.
Screenshot 2023-10-23 at 12 36 32

**Review by vadym-zinchenko-moc — CHANGES_REQUESTED** [review]

**Comment by vadym-zinchenko-moc** [comment]
Fix these small issues and I will approve! Very nice work! Thx!

File: homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html (hunk 2 above)
Comment (lesiaturuta) [reply homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html:24]: changed the distance and made header 100%

File: homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html (hunk 1 above)
Comment (lesiaturuta) [reply homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html]: added links

**Review by vadym-zinchenko-moc — APPROVED** [review]

**Pull request merged**